spec/parallel_tests/cli_spec.rb

```diff
@@ -149,6 +149,36 @@ def call(*args)
       end
     end
   end
+
+  describe "#display_duration" do
+
+    def call(*args)
```

Comment:
good point, things like call/run can overwrite something by accident and
then you are in some hard to debug mess :)

On Wed, Feb 25, 2015 at 4:28 PM, Ed Slocomb <notifications@github.com>
wrote:

> In spec/parallel_tests/cli_spec.rb
> <https://github.com/grosser/parallel_tests/pull/366#discussion_r25394454>:
>
> > @@ -149,6 +149,36 @@ def call(*args)
> >        end
> >      end
> >    end
> > +
> > +  describe "#display_duration" do
> > +
> > +    def call(*args)
>
> fwiw, I don't like naming a method call() unless I'm explicitly overriding
> ruby's Method#call. Naming something else call() can potentially ruin my
> day if I come along later and do functional programming tricks,
> instrumentation tools, or other dark arts.
>
> But when in Rome... ;)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grosser/parallel_tests/pull/366/files#r25394454>.
>